projboard/tests/test_article.py

```diff
@@ -1,47 +1,13 @@
 import pytest
-from projboard.models.article import Article, Like, View_Article, Subject, User
+from projboard.models.article import Article, Like, View_Article, Subject
 from django.db.models import Count
 
 
-TITLE = "Article Test"
-CONTENT = "I'm not gonna write all that!"
-
-
+@pytest.mark.django_db
 class TestArticle:
 
-    @pytest.fixture
-    @pytest.mark.django_db
-    def create_subjects(self):
-        sup = Subject(name='N')
-        sup1 = Subject(name='R')
-        sup.save()
-        sup1.save()
-        return [sup, sup1]
-
-    @pytest.fixture
-    @pytest.mark.django_db
-    def create_user(self):
-        user = User(name="Anas", nickname="A99", email="Anas@gmail.com", password="Anas123")
-        user1 = User(name="John", nickname="J99", email="John@gmail.com", password="John123")
-        user.save()
-        user1.save()
-        return [user, user1]
-
-    @pytest.fixture
-    @pytest.mark.django_db
-    def create_articles(self, create_user, create_subjects):
-        articles_list = []
-        for i in range(len(create_user)):
-            article = Article(user_id=create_user[i], title=TITLE,
-                              content=CONTENT, subject_id=create_subjects[i])
-            article.save()
-            articles_list.append(article)
-
-        return articles_list
-
-    @pytest.mark.django_db
-    def test_part_functions(self, create_articles):
-        article = create_articles[0]
+    def test_part_functions(self, articles):
+        article = articles[0]
```

Comment:
Maybe can we use here the article fixture instead articles fixture?
I think we use only one article in this test.

@Anas-mhameed @rawadabu WDYT?

Reply:
Consider to change to `def test_part_functions(self, article)`.
Also remove line 10, i think no need to change the other lines because they will use the article fixture.

Reply:
Exactly, done, thank you @Shalevro2

Reply:
Great!